tests/setup.ts

```
@@ -15,11 +15,12 @@ globalThis.foundry = {
   applications: {
     api: {
       ApplicationV2: class {
-        render = vi.fn();
+        async close(_options = {}) {}
       },
-      HandlebarsApplicationMixin: (base: any) => base,
+      HandlebarsApplicationMixin: (Base: any) => class extends Base {},
     },
   },
+  utils: {},
```

Biome (2.4.7)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)